app/routes/forms_rst.py

```diff
@@ -0,0 +1,27 @@
+from http.client import HTTPException
+
+from discord_webhook import DiscordWebhook
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+from app.common.config import DEMO_DISWEBHOOK_URL
+
+
+class DemoFormSchema(BaseModel):
+    name: str
+    contacts: list[str]
+
+
+router = APIRouter(tags=["forms"])
+
+
+@router.post("/demo/", status_code=204, summary="registration on demo")
```

Comment:
question: Is it necessary to consider this endpoint as demo? I haven't seen any info about demo in Kaiten